packages/sdks/e2e/react/src/App.tsx

```diff
@@ -54,6 +54,18 @@ function App() {
     }
   }, []);
 
+  useEffect(() => {
```

Comment:
I think the better way would be to add unit test in `generate-content-url.test.ts` instead of useEffect here. The aim is to respect empty string `''` passed to `omit` field.

@samijaber or @sidmohanty11 can correct me on this.

Reply:
The snapshots also should be updated to differentiate omit as undefined vs omit as empty string.
ie `generate-content-url.test.ts.snap`